extensions/kind/scripts/download.ts

```diff
@@ -62,6 +62,7 @@ async function download(tagVersion: string, repoPath: string, fileName: string):
   });
   let buffer;
 
+  if (!manifests.data.content) return;
```

Comment:
if this is an expected case, we should throw an error instead of simply return, like if this is something that should never happen, better to throw